test/files.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+
+module.exports = (common) => {
+  describe('.file', () => {
+    let ipfs
+
+    before((done) => {
+      common.setup((err, _ipfs) => {
+        expect(err).to.not.exist
+        ipfs = _ipfs
+        done()
+      })
+    })
+
+    after((done) => {
+      common.teardown(done)
+    })
+
+    describe('.cat', () => {
+      it('returns file stream', (done) => {
+        const hash = 'QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o'
```

Comment:
right now you can use a hash for the 'init docs' that gets bundled with IPFS. But add a note saying to replace that hash by something you put with ipfs.add, once that is ready.